src/scripts/modules/gradebook.js

```diff
@@ -48,6 +48,9 @@
     ];
 
     const PARSING_TIME_FORMAT = "MMM DD, YYYY hh:mm A";
+    const slice = c => [].slice.call( c );
+    const qs = ( el, s ) => el.querySelector( s );
```

Comment:
This is a vestige of the "keas" implementation. This simply moved existing functions from the scrape method. 

100% right, as we now have jquery as a dependency, just haven't gone through and done the work.